bears/c_languages/CPPLintBear.py

```diff
@@ -26,14 +32,22 @@ class CPPLintBear:
     def create_arguments(filename, file, config_file,
                          max_line_length: int=79,
                          cpplint_ignore: typed_list(str)=(),
-                         cpplint_include: typed_list(str)=()):
+                         cpplint_include: typed_list(str)=(),
+                         use_spaces: bool=True,
+                         indent_size: int=2):
```

Comment:
We use a default indent of `4` in space consistency IIRC, maybe we should go with `4` here as well